packages/plugins/rrweb-plugin-console-record/test/stringify.test.ts

```diff
@@ -6,7 +6,28 @@ import { stringify } from '../src/stringify';
 import { describe, it, expect } from 'vitest';
 
 describe('console record plugin', () => {
+  const aLongString = 'a'.repeat(100);
```

Comment:
```suggestion
  const aLongString = 'a'.repeat(100) + 'b';
```
(Ensure that we are testing that we're removing the end of the string and not the start)